src/LensPlatformClient.test.ts

```diff
@@ -75,14 +71,17 @@ describe("LensPlatformClient", () => {
       ];
 
       const _got = lensPlatformClient.got;
-      await Promise.all([
-        _got.get(apiEndpointAddress),
-        _got.post(apiEndpointAddress),
-        _got.put(apiEndpointAddress),
-        _got.patch(apiEndpointAddress),
-        _got.head(apiEndpointAddress),
-        _got.delete(apiEndpointAddress)
-      ]);
+
+      try {
+        await Promise.all([
+          _got.get(apiEndpointAddress),
+          _got.post(apiEndpointAddress),
+          _got.put(apiEndpointAddress),
+          _got.patch(apiEndpointAddress),
+          _got.head(apiEndpointAddress),
+          _got.delete(apiEndpointAddress)
+        ]);
+      } catch (e: unknown) {}
```

Comment:
Maybe put the "spies tests" in the finally block? Or add a comment to explain why we want to catch the error and ignore it here. (non trivial for me when reading here) 

```suggestion
      } finally {
        spies.forEach(spy => {
          // tests are moved here
        }
      }
```